Makefile

```diff
@@ -16,6 +16,11 @@ export GOPATH=$(shell go env GOPATH)
 # WPTD_PATH will have a trailing slash, e.g. /home/jenkins/wpt.fyi/
 WPTD_PATH := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 WPTD_GO_PATH ?= $(GOPATH)/src/github.com/web-platform-tests/wpt.fyi
+WEBDRIVER_PATH ?= $(WPTD_GO_PATH)/webdriver
+BROWSERS_PATH ?= $(HOME)/browsers
+SELENIUM_PATH ?= $(BROWSERS_PATH)/selenium-server-standalone-3.8.1.jar
```

Comment:
Do we want to include the versions here? Might make maintenance easier if we rename all of these to not include the version number and then this script can be update invariant.

Reply:
Removed.